Cn video patch 2

wjllance opened the pull request.

AndrewJakubowicz commented on app/views/courses/CourseVideosComponent.vue:
```diff
@@ -31,7 +39,11 @@
 
       .row.m-t-5
         .col-md-6
-          iframe.video(src="https://player.vimeo.com/video/310626807" frameborder= "0" webkitallowfullscreen mozallowfullscreen allowfullscreen)
```
Just wanted to highlight that I really like that you're grabbing the video urls from the `utils.coffee`.
Thanks for improving the code.

wjllance commented:
> Didn't manually test but looks like it addresses code reviews.
> Great stuff @wjllance !!! 👍

Thanks! Maybe we can have more test later on with China team:)
